include/effects/matrix/PatternMaze.h

```diff
@@ -294,9 +294,13 @@ class PatternMaze : public LEDStripEffect
 
         cellCount = 0;
         hue = 0;
-        g()->Clear();
```

Comment:
Can't talk to the graphics API yet in effect Init, this only worked before by chance!